Coalition: Expeditions Past (#6324)
endless-sky/endless-sky: data/coalition/coalition missions.txt

			`	After he shakes your hand, you say your goodbyes and board your ship.`
				decline

mission "Coalition: Expeditions Past 1"
	minor
	name "Scouting Permit"
	description "Transport Sedlitaris, a Heliarch Tribune, to the <destination>, where she will try to petition for the Heliarch council to allow her to go on a scouting expedition out of the Coalition. Payment is <payment>."
	passengers 1
	to offer
		"coalition jobs" >= 40
		random < 5
	source
		government "Coalition"
	destination "Ring of Friendship"
	on offer
		require "Jump Drive"
		conversation
			`As you exit your ship and start heading to the spaceport, you hear a rushed galloping getting closer, and turn to see a Saryd Heliarch coming to you, gesturing for you to wait. She slows down once she sees you're obliging, and slowly walks to you, while glancing at your ship.`
			`	"Apologize for cutting you off, I must, Captain <last>, but in need of extended transport services, I am," she says. "Sedlitaris Quatcid, my name is, a Heliarch tribune. Brought to <destination>, I must be. A petition to the council, I am to make."`
			choice
				`	"A 'petition?' What do you mean?"`

> 				`	"I don't think you'd need some 'extended' service just for that."`

Author: She tells you she'll need your services for a while, but at first just tells you she wants transportation to the ring, this is just another choice to inquire about what she's up to.
Basically "I don't think you'd say you needed my help for a long time if you only wanted me to transport you there, what else will I have to do?" is what I tried to get there, might be an awkward line I guess, don't know.

> 				`	"I don't think you'd need some 'extended' service just for that."`

Reviewer: Ah ok. Makes sense now.

				`	"Sorry, I'm not taking any more passengers right now."`
					decline
			`	"Nearly a thousand years ago, leading an expeditionary fleet to outside of the Coalition, an ancestor of mine was." She explains. "Mimodum, his name was, and after having risen the ranks and lifted our family from poorer conditions, chosen to lead the expedition he was. Only, return, he did not. To find out what happened, I wish. Looking for you, I have, as after brought to <planet>, I have been, and approved, my request is, need a transport to bring me to planets he visited, I will."`
			choice
				`	"I'd be glad to help you find out what happened. Let me show you to your bunk."`
					goto accept
				`	"Couldn't you just have gone on your own, or paid someone to bring you there if you don't have a ship?"`
				`	"Actually I'm not going to be leaving Coalition space anytime soon, sorry."`
					decline

> 			`	She shakes her head. "Forbidden from leaving, anyone is. Steal a jump drive for that, I'd also need to do, and obviously illegal, that is. Allowed to head out, only Heliarchs who have received permission from the council can. Finally time to hear my request again, they have, so go to <planet>, I must."`

Author: The Heliarchs have stopped sending ships out because as the missions reveal all the latest expeditions never returned, so it's a waste of Jump Drives to be sending them out, as they see it. They only stick to the occasional visits to Poisonwood, which is just one jump away and relatively safe.
Sedlitaris in particular has been trying to get this permission for a while, and due to her insistence they revoked her rights to a Heliarch ship, just in case she tried something like taking a Jump Drive by force with said ship. She's hiring you because she knows the Heliarchs won't provide her a Jump Drive, much less a fleet.

I thought I had pointed to that enough, I don't know where exactly you'd like me to explain it more? Just comment if you have a specific part of the mission in mind for that.

> 			`	She shakes her head. "Forbidden from leaving, anyone is. Steal a jump drive for that, I'd also need to do, and obviously illegal, that is. Allowed to head out, only Heliarchs who have received permission from the council can. Finally time to hear my request again, they have, so go to <planet>, I must."`

Reviewer: I'm sorry, I made this comment before I read the entire mission. It makes sense in retrospect. Feel free to resolve.

	passengers 6
	blocked "You need <capacity> to take the next mission. Return here when you have the required space free."
	to offer
		has "Coalition: Expeditions Past 1: done"
	source "Ring of Friendship"
	destination "Ablub's Invention"
	on offer
		conversation
			`Looking for a place to sit by the entrance to the spaceport, you spot a set of seats made for the Quarg, still around to this day. You find one that you assume was meant for a Quarg child, standing lower than the others, and wait. Captains and workers that pass by give you all sorts of looks, and after a few minutes even some Heliarch agents are gawking at you, discussing among themselves. Just as you think if you should get up to sit somewhere else, look to see Sedlitaris coming into the spaceport.`
			`	"A short wait, wasn't it?" She comments once you're close. "Care to hear my case, the consuls didn't, just told me that to leave whenever I wish, I may. 'If I can,' they said, as provide me a jump drive, they won't, but that's where you come in."`
			choice

> 				`	"I certainly didn't wait around for long. So, are we heading for human space?"`

Author: It's currently the only place one can go to from the Coalition, but more areas might be added around so mentioning human space specifically won't work in the long run.
I'll change it in a bit.

					goto heading
				`	"Didn't care to hear your case? What do you mean?"`
			`	"According to protocol, present valid reasoning for their expedition, aspiring inspectors must. This time, taken every legal detail into account, I had, in preparation for the meeting, but once I entered, barely time to greet them, they gave me." She explains. "Said that thought it over and willing to oblige, they were, so just sign a few documents, I did, and now officially an inspector, I am."`
			choice
				`	"I take it 'inspector' is what the people sent out are called?"`

> 				`	"Good to hear they had a change of heart. So, are we heading for human space?"`

Author: No, I just forgot I had the option in two `choice` instances.

			label heading
			`	"Ready to go outside, we are not. Not for the purpose of finding out about the last expedition, at least. A tool from House Idriss, we need, if to trace my ancestor's steps, we are, so to <destination>, we must go."`
			`	As she's about to continue, your conversation is cut short by a Kimek rushing towards you two, shouting frantically. Once he's closer, you recognize him as arbiter Kicharri from earlier, and Sedlitaris tries to calm him down while he's catching his breath. "Professor, if about my request this is, already met with the council, I have. Approved it--"`
			`	"I know! Exactly what I'm furious about, that is!" He shouts. "Asked them to give you a definitive answer, I had, but not this kind of definitive! Listen, permission or not, safe to leave, it is not. Know that, you do." He looks at her for a few moments, then to you. "Captain, as her superior, greatly appreciate your intent to help Sedlitaris, I do, but too brash about this, she has been. Take her out of our space, you must not."`

> 			`	He looks at her again, as if expecting some response. After a fez seconds, he turns away slowly, saying "Don't waste your life on this," and walking away. Sedlitaris sighs once he turns a corridor, and says she's ready for the trip.`

Reviewer 2: ```suggestion
			`	He looks at her again, as if expecting some response. After a few seconds, he turns away slowly, saying "Don't waste your life on this," and walking away. Sedlitaris sighs once he turns a corridor, and says she's ready for the trip.`
```

			label next
			`	She compares the coordinates for Ancient Light with your map, and finds <destination>. You tell her you'll set course for <planet>, and she heads back to her room.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris hasn't entered the system yet. Better depart and wait for it.`

mission "Coalition: Expeditions Past 6"
	landing
	name "Report: Summit"
	description "Take Sedlitaris to <destination>, the furthest point north on the pylon's records, and next stop of the expeditionary fleet."
	passengers 1
	to offer
		has "Coalition: Expeditions Past 5: done"
	source "Skymoot"
	destination "Watcher"
	on offer
		conversation
			`Sedlitaris starts triangulating to find the next pylon again, and brings you to a tall mountain, where some of the local dragons appear to have made some form of nest or breeding ground. You do your best to avoid them as you fly around the mountain, and Sedlitaris continues to send signals to find the pylon.`

> 			`	"Difficult to believe it, I find, that anyone would attempt to dig in a home to the dragons, but maybe inside the mountain, it is. From the mountain, the countersignals are coming." She stops for a few minutes, and looks at the view of the mountain on your monitors. She tries again, and asks that you get closer to the mountain. When you're finishing yet another lap, she asks that you stop your ship, keeping it steady in the air. "Zoom your cameras here, could you, Captain?" she asks, pointing to the middle of a monitor. You zoom in and see a circle of large rocks, surrounded by dragon nests. Atop the rocks are some patches of foliage, and in the middle lies a small, dark, tower-like structure, stuck in the rocks by one end. After looking at it for a few seconds, it clicks for the two of you that it's an upside-down pylon. "Taken it as some form of token, the dragons must have," Sedlitaris says. "Hopes of retrieving some of the pylons physically, I had, but prefer to dig out the glacier for the previous one than test my luck against the dragons, I do. Unlock this one's data remotely, I will."`

Reviewer: ```suggestion
			`	"Difficult to believe it, I find, that anyone would attempt to dig in a home to the dragons, but maybe inside the mountain, it is. From there, the countersignals are coming." She stops for a few minutes, and looks at the view of the mountain on your monitors. She tries again, and asks that you get closer to the mountain. When you're finishing yet another lap, she asks that you stop your ship, keeping it steady in the air. "Zoom your cameras here, could you, Captain?" she asks, pointing to the middle of a monitor. You zoom in and see a circle of large rocks, surrounded by dragon nests. Atop the rocks are some patches of foliage, and in the middle lies a small, dark, tower-like structure, stuck in the rocks by one end. After looking at it for a few seconds, it clicks for the two of you that it's an upside-down pylon. "Taken it as some form of token, the dragons must have," Sedlitaris says. "Hopes of retrieving some of the pylons physically, I had, but prefer to dig out the glacier for the previous one than test my luck against the dragons, I do. Unlock this one's data remotely, I will."`
```

			`	A dragon perches on your ship as she begins the transfer. It tries to bite down on the engines, but you start moving your ship again to shake it off. You avoid other dragons as they fly by your ship, until the download is complete, and you head straight for the spaceport city.`

> 			`	Once your ship has safely landed on the hangars, Sedlitaris plays the audio log: "Log started. Scoured the planet with our searches, we have, but no trace of the previously visitted ruins, or any other sign of the group that lived here, we found. Assured me that all our ships' navigation systems are working proper, the engineers have, but at the coordinates where they should be, the ruins are not, as if they had never been there. A confusing disappointment, this was, but continue with the mission, we must. Heading far north, to Summit, we are. The last point before regrouping with the split group, it will be."`

Reviewer 3: "visitted" typo.

			label end
			`	You open your map, and the coordinates point to "Summit" being <planet>, a small moon close to the entrance to The Deep. Sedlitaris returns to her bunk, and you set course to <system>.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris hasn't entered the system yet. Better depart and wait for it.`

mission "Coalition: Expeditions Past 7"
	landing
	name "Curious Red Cloud"
	description "Look for the system near Algieba where Mimodum, the expedition's leader, jumped to investigate the cosmological phenomenon in the system."
	passengers 1
	to offer
		has "Coalition: Expeditions Past 6: done"
	source "Watcher"
	on offer
		conversation
			`You fly around the moon as Sedlitaris nails down the coordinates of the pylon, until you land close to one of the forests. "Deeper inside, the pylon must be. Uninhabited by humans this world is, correct?" You confirm it, and she prepares to leave. "No giant dragons to worry about here, there are, so try to bring back this pylon with us, I will."`

> 			`	You two leave your ship and enter the forest, taking turns carrying the pylon you brought, which is easy enough to carry due to the lower gravity. Occasionally Sedlitaris stops and has the pylon put down so that she can determine where to go next. Eventually, you notice the trees start getting smaller as you move on, until you reach a wide, odd clearing where the ground is devoid of any of the local flora. Sedlitaris spots the other pylon on the opposite side, half stuck in the ground, and moves to pull it out. As you follow her to the machine, you notice deep, perfectly straight holes in the ground, forming lines that sometimes cross the entire clearing and continue well past it. Soil and dust have filled most of them up, but in certain gaps you can look down to cracks that must be dozens of meters deep.`

Reviewer: ```suggestion
			`	You two leave your ship and enter the forest, taking turns carrying the pylon you brought, which is easy enough to do since the gravity is so much lower here. Occasionally Sedlitaris stops and has the pylon put down so that she can determine where to go next. Eventually, you notice the trees start getting smaller as you move on, until you reach a wide, odd clearing where the ground is devoid of any of the local flora. Sedlitaris spots the other pylon on the opposite side, half stuck in the ground, and moves to pull it out. As you follow her to the machine, you notice deep, perfectly straight holes in the ground, forming lines that sometimes cross the entire clearing and continue well past it. Soil and dust have filled most of them up, but in certain gaps you can look down to cracks that must be dozens of meters deep.`
```

			`	Sedlitaris ties a rope around the end of the pylon that's visible, and pulls it, forcing up the dirt as she brings the rest of the device to surface. She connects it with the pylon you brought, and acesses the data from there. "Much more active here, the expedition was," she says. "Three recordings, there are. Bring these back to your ship, we should, listen to the first one there, we can."`

> 			`	Each of you carrying a pylon, you make your way out of the forest and back to your ship, where Sedlitaris immediately plays the oldest entry: "Log started. Sent two Punishers further north, I have, but returned yet, they have not. As for the crimson anomaly on the nearby western system, still unresponsive, it is. A reaction from it, nothing got, be it a Finisher barrage or attempting to ram it. A simple nebula, the researchers claim it is not, think that we are missing something, they do. Wait a few more days for the Punishers to return, I will. In the meantime, back to that system, I will go. Maybe learn something about the anomaly, we can."`

Reviewer: ```suggestion
			`	Both of you carrying a pylon, you make your way out of the forest and back to your ship, where Sedlitaris immediately plays the oldest entry: "Log started. Sent two Punishers further north, I have, but returned yet, they have not. As for the crimson anomaly on the nearby western system, still unresponsive, it is. A reaction from it, nothing got, be it a Finisher barrage or attempting to ram it. A simple nebula, the researchers claim it is not, think that we are missing something, they do. Wait a few more days for the Punishers to return, I will. In the meantime, back to that system, I will go. Maybe learn something about the anomaly, we can."`
```

					goto more
			`	She doesn't respond, instead looking down, at the lines carving the clearing's ground. She moves close to one, laying down and reaching her arms to touch the cracks. A few minutes later, she gets back up, and looks to you.`
			label more
			`	"Back to the ship, we should go," she says. "Try and figure out where Mimodum went next, I will. Where he was to meet with the split group."`
			`	After taking the trail back, Sedlitaris goes into her bunk to tinker with the pylons. She comes out hours after nightfall, asking that you show her your map. "A pincer movement, the two groups were performing. Difficult to know the exact planet, it is, as the data from the split group, we don't have, but figured the overall region where the two groups were link up, I think I have."`
			`	You help her match the coordinates, and end up with a group of systems between Sol and Tarazed. "Need to visit each system, we might. Pick up faint signals of their next pylon, while in the right system I can, but no way of knowing where that is exactly, I have."`
				accept
	on enter "Rasalhague"

> 		dialog `Sedlitaris has the pylon start searching for any matching signals as soon as you jump into the system. A few hours pass, and the machine doesn't return any coordinates, however it also doesn't report a failure. Sedlitaris looks at it puzzled, and reboots it. "Fly around the system for a while, could you, Captain? Maybe compromising the signal, the ship's current position is." You do as she suggests, and slowly cruise around the system for a few more hours. Once again, the pylon doesn't give any answer. Sedlitaris says it might be some oddly specific bug, and asks that you just move on to the next system.`

Reviewer: ```suggestion
		dialog `Sedlitaris has the pylon start searching for any matching signals as soon as you jump into the system. A few hours pass and the machine doesn't return any coordinates, but it also doesn't report a failure. Sedlitaris looks at it puzzled, and reboots it. "Fly around the system for a while, could you, Captain? Maybe compromising the signal, the ship's current position is." You do as she suggests, and slowly cruise around the system for a few more hours. Once again, the pylon doesn't give any answer. Sedlitaris says it might be some oddly specific bug, and asks that you just move on to the next system.`
```

			`	As you're flying to the hangars, she plays the pylon's entry: "Log started. Made a route from Summit straight to the meeting point, I had, but find the split group, I could not. Restless, agitated, the crew has become, heading straight to Ablub, we are, after buried, this thing is. If find this, anyone from the split group is, head back-" at that point an untranslated shouting muffles what Mimodum was saying, followed by dozens of legs in a rush. "Oh no..." is heard right before the recording ends.`
			`	You focus on flying the ship, but even when you've landed, Sedlitaris remains silent.`
			choice
				`	"Are you alright?"`
				`	"He said they were going back immediately, but they never made it. You think that was the last pylon?"`
				`	(Stay quiet).`
			`	She goes without a word for a while still. "Set a route for Ablub, could you, Captain? If follow the same route they used, we do, and if any more to find, there is, pick up the signals, I will."`
			`	You agree, and look to set a route straight to Ablub.`
				accept
	on enter "Girtab"

> 		dialog `While Sedlitaris remained in her bunk since you left for Ablub, you can hear her shouting from her bunk as you enter the system. She comes running to you, bringing along the pylon. "Picked up a signal, it has!" She says, pointing at the machine's panel, where some message is repeatedly beeping in and out. "Land on that planet we must, Captain!" You tell her you'll land soon, and she remains with you to figure out the pylon's location on Harmony, once you've landed.`

Reviewer: ```suggestion
		dialog `While Sedlitaris remained in her bunk since you left for Ablub, you can hear her shouting from her bunk as you enter the system. She comes running to you, bringing along the pylon. "Picked up a signal, it has!" She says, pointing at the machine's panel, where some message is repeatedly beeping in and out. "Land on that planet we must, Captain!" You tell her you'll land soon, and she remains with you to figure out the pylon's location on Harmony once you've landed.`
```

				goto accept
			label alot
			`	"Ah, forgive me for that, Captain. True it is, that disappointed, I am. However, an inspector should not let that trouble her duties," she explains. "Done your work, you have, so now do mine, I must. Not talked about in Mimodum's audio entries, many details of interest to the council were. An initial gathering of that data, I have prepared."`
			label accept
			`	She asks that you bring her to <destination> as soon as possible, and goes back to her bunk to rest.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris hasn't entered the system yet. Better depart and wait for it.`
	on complete
		"coalition jobs" += 15
		payment 35000000
		conversation
			`Sedlitaris uses your commlink to tell the spaceport authorities that she's returned, and soon after you land a crowd of Heliarchs and Coalition civilians alike are surrounding your ship. The Heliarch agents try and organise the crowd as best as they can, and a group of Heliarch consuls rush through the masses to meet you two at your ship.`

> 			`	"Inspector! A most... fortunate return this is. A great deal of information you've collected, we take it?" An Arach consul asks.`

Reviewer: ```suggestion
			`	"Inspector! A most... fortunate return this is. A great deal of information you've collected, we take it?" an Arach consul asks.`
```

> 			`	"Yes, eager to hear your report, we are," one of the Kimek consuls continues. "A rare occasion, it is, to be able to learn of the outside from one of our esteemed inspectors"`

Reviewer: ```suggestion
			`	"Yes, eager to hear your report, we are," one of the Kimek consuls continues. "A rare occasion, it is, to be able to learn of the outside from one of our esteemed inspectors."`
```

				goto end
			label forgive
			`	She breathes in relief. "Thank you, Captain. To trick you, my intention wasn't. Thanks to your help, maybe put an end to this fear of leaving, or even convince the council to get the expedition efforts going again, I can."`
			label end
			`	Suddenly, a Saryd Sojourner nearly rams your ship, just barely avoiding it as it comes to a stop while turning. "My little brother," Sedlitaris says as she looks at the ship. "Forgive me Captain, but get to seeing my family, I must. From the discussion earlier, very worried about me, they must have been. Thanks again for everything." Her brother rushes into hugging her as he leaves his ship, then proceeds to shake her while screaming something. An elderly Saryd couple slowly comes out of the Sojourner as well, and Sedlitaris and her brother go and meet them, hugging and going inside. The Sojourner takes off before the spaceport workers can even begin tending to the ship, and you look at it as it leaves the ringworld.`

> 			`	When you go back inside your ship, you're notified that more credits have been transferred to your account. It seems to be from the Heliarch government, giving you an extra 25,000,000 credits, thanking you for helping recover the data of the sixth.`

Reviewer: ```suggestion
			`	When you go back inside your ship, you're notified that more credits have been transferred to your account. It seems to be from the Heliarch government, giving you an extra 25,000,000 credits, thanking you for helping recover the data of the sixth expedition.`
```

> 			`	When you go back inside your ship, you're notified that more credits have been transferred to your account. It seems to be from the Heliarch government, giving you an extra 25,000,000 credits, thanking you for helping recover the data of the sixth expedition.`

Reviewer: ```suggestion
			`	When you go back inside your ship, you're notified that more credits have been transferred to your account. It seems to be from the Heliarch government, giving you an extra 25,000,000 credits and thanking you for helping recover the data of the sixth expedition.`
```

	to offer
		has "Coalition: Expeditions Past 1: done"
	source "Ring of Friendship"
	destination "Ablub's Invention"
	on offer
		conversation
			`When you don't see Sedlitaris in the spaceport, you look for a spot to sit and wait for her, and notice a set of seats made for the Quarg, still around to this day. You take one that you assume was meant for a Quarg child, standing lower than the others, and wait. Captains and workers that pass by give you all sorts of looks, and after a few minutes even some Heliarch agents are gawking at you, discussing among themselves. Just as you think if you should get up to sit somewhere else, you look to see Sedlitaris coming into the spaceport.`
			`	"A short wait, wasn't it?" she comments once you're close. "Care to hear my case, the consuls didn't, just told me that to leave whenever I wish, I may. 'If I can,' they said, as provide me a jump drive, they won't, but that's where you come in."`
			choice
				`	"I certainly didn't wait around for long. So, where are we heading?"`
					goto heading
				`	"Didn't care to hear your case? What do you mean?"`

> 			`	"According to protocol, present valid reasoning for their expedition, and a thorough travel plan, aspiring inspectors must. This time, taken every legal detail into account, I had, in preparation for the meeting, but once I entered, barely time to greet them, they gave me." She explains. "Said that thought it over and willing to oblige, they were, so just sign a few documents, I did, and now officially an inspector, I am."`

Author: It's supposed to be implied by later conversations that they just got annoyed by her and decided to send her off to die, as they didn't think she would come back, just like previous expeditions didn't.

			choice

> 				`	"I take it 'inspector' is what the people sent out are called?"`

Author: Just figured this particular rank warranted a specific title, I don't remember if inspector was suggested by someone or the exact reasoning for it if it was me, likely just boils down to "it sounded cool" though.
I assume the three Coalition languages aren't 1:1 with one another, or with human languages, so they'd have to have approximations for words that don't really exist on one language or the other. Would just be an implied approximation I guess.

> 				`	"I take it 'inspector' is what the people sent out are called?"`

Reviewer 4: Got it. If `investigator` is more of a title/rank then, I'd think it should become a proper noun. Basically, just convert all `'investigator'`s to `Investigator`. Makes it more clear this is a specific rank or role we're referring to.

> 				`	"I take it 'inspector' is what the people sent out are called?"`

Author: Removed the quotes, added an extra bit at the start of her response to give more of an explanation on the rank/title.
Let me know if it still needs work.

			`	He looks at her again, as if expecting some response. When no response comes, he turns away slowly, saying "Don't waste your life on this," and walking away. Sedlitaris sighs once he turns a corridor, and says she's ready for the trip.`
			choice
				`	"Hold on, what did he mean by it not being safe to leave? I thought you had permission."`
				`	"Alright, let's head back to the ship."`
					goto end
			`	"I do. Know what he meant, I also do not," she says, looking back to the corner Kicharri turned. "Approving of my wish for this trip, he never was. Some time to calm down, he needs. For now, back to your ship, we should go."`
			label end
			`	You find five Heliarchs waiting by your ship, who Sedlitaris introduces as representatives of the council, and who must be present for when House Idriss provides the piece of equipment she wants. You show her and the others to their bunks and prepare for the trip.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris and the Heliarch council representatives hasn't entered the system yet. Better depart and wait for it.`

> mission "Coalition: Expeditions Past 3"

Author: I see the pylon as effectively the same as the Quarg device that they permit Elias to use in helping you locate Giftbringer.
As in, it's a tech piece that's not granted to the player, but to someone trusted by the faction of the person the player will be transporting. Similarly, I imagine both Elias and Sedlitaris keep the devices in their bunks when not in use.
The Quarg tracking thing doesn't have an outfit for it, so I don't think one is needed for the pylon here.

> mission "Coalition: Expeditions Past 3"

Reviewer 4: Do you think it's big enough to warrant cargo space?

> mission "Coalition: Expeditions Past 3"

Author: Probably would take up space that could otherwise be filled with a cargo crate or two, but it's not something kept in your cargo bay, so I don't see the need for it.

			`	"Repilor, a work trip, this is. Away for long, I won't be, so wish to trouble you all with the news, I didn't," she responds.`
			`	"Trouble us? Truth is, know that wrong, you are, but face telling father that you're throwing your life away, you couldn't!"`
			`	He tries to push his way to her, but the guards stop him before he can make much progress. One of the Heliarch representatives gets closer, saying, "A restricted area, this is. Invading government property, you are, so suggest that you leave immediately, I do."`
			`	"Restricted? What about the outside? Is it not also restricted? Much more dangerous? Fine with sending my sister there, you seem," Repilor counters, and the Heliarch looks away after a moment. "If bothering the consuls so much, she was, just fired, she should've been!" He then looks at you, and continues, "And the human? Your transport out of here, I imagine? Or maybe the one who put these ideas in your head, they are?"`
			choice
				`	"I've just been hired to transport her."`
					goto hired
				`	"What's so wrong about her wanting to explore outside the Coalition?"`

> 			`	"Awaiting 'explorers,' nothing but an unknown, certain doom is. Wish for that to be my sister's fate, I do not," he answers.`

Author: In this he's responding to the player's prompt about "what's wrong with wanting to **explore**" so I felt it'd be appropriate to have him use a word closer to that (explorers) in his reply.
If you think that gets inconsistent because of the rank name, could change it so that he doesn't have the quoted portion within his reply.

> 			`	"Awaiting 'explorers,' nothing but an unknown, certain doom is. Wish for that to be my sister's fate, I do not," he answers.`

Reviewer 4: Makes sense, the use of single quotes is just what threw me off. With the change of `Inspector` to a proper noun (see above unresolved conversation on line 2412), this should be fine.

			choice
				`	"How do you know about that species?"`
				`	"What was in the ruins before they disappeared?"`
			`	"Found the ruins millennia ago, the first Saryds to come to the outside did. Recovered some artifacts, they also had, complex tools, such as very large, personal propelling devices, with dedicated storage space. Kept in some museums back in Saros, they are."`
			choice
				`	"Did they ever find any member of that species?"`
				`	"They couldn't find out anything more? Just some jetpacks with pockets?"`
			`	She shakes her head. "Stocked primarily with provisions, those first expeditions were. Prepared to take on proper archaeological study on this planet, they were not. Asked the Quarg about the findings, the Saryds had, but to their surprise they acted as if there was nothing here. For the time, very odd of them to do that, it was." She turns to you with a curious look. "What about humanity, Captain? Found any clues about the species that lived here, have you?"`

> 			branch sheragi

Author: Like what? An increase in pay?

			`	Sedlitaris ties a rope around the end of the pylon that's visible and pulls it, forcing up the dirt as she brings the rest of the device to surface. She connects it with the pylon you brought, and accesses the data from there. "Much more active here, the expedition was," she says. "Three recordings, there are. Bring these back to your ship, we should, listen to the first one there, we can."`
			`	Both of you carrying a pylon, you make your way out of the forest and back to your ship, where Sedlitaris immediately plays the oldest entry: "Log started. Send two Punishers further north, I did, but returned yet, they have not. As for the crimson anomaly on the nearby western system, still unresponsive, it is. A reaction from it, nothing got, be it a Finisher barrage or attempting to ram it. A simple nebula, the researchers claim it is not, think that we are missing something, they do. Wait a few more days for the Punishers to return, I will. In the meantime, back to that system, I will go. Maybe learn something about the anomaly, we can."`

> 			`	After Mimodum's voice has disappeared, Sedlitaris urges you to take off. "Heard of this 'anomaly' before, I had not. Look for the system it's in, at once we must. Hopefully still there after the centuries, it is."`

Author: It was meant to be more a gameplay thing yes. Either you don't know where Terminus is, and have to pay attention to what the recording says to find it, or you'll know what they're talking about from the description of the anomaly.
Also, potentially something to serve as a pointer to Terminus in case one hasn't found it yet.

			`	"Tried just about everything now, we have. A finisher barrage, bombardment from afar, driving asteroids through it, even activating our jump drive while immersed in the red cloud. Told me that it shows some signs of rotation, the chief researcher has, tried to show me the axis around which it spins, she did. Whatever the case, find anything worthwhile about the cloud, we could not. Returning to Summit now, we are, and from there to regroup with the fleet."`
			`	The recording ends, but Sedlitaris continues looking out to the "cloud" for a few minutes, in silence.`
			choice
				`	"Are you going to play the last one?"`
					goto last
				`	"Is something wrong?"`
			`	"No, just... taking in the view, I was. Unlike any phenomena from within the Coalition, this is."`
			label last
			`	She moves back to the pylon, and hesitates for a moment. "Back to Summit, Mimodum said they would go. Better understand his next steps, we would, if listen to the last entry on that clearing where we found this, we did," she says, tapping the pylon.`
			`	You tell her you'll go back to <origin> soon, and she heads to her bunk.`

> 		fail

Author: I haven't tested that, will need to check.

> 		fail

Author: Tested it. Doesn't seem to work.
Thing is the mission automatically offers and accepts alongside the main one ("Curious Red Cloud"), so `decline` just forces an endpoint to the conversation. The mission was already accepted when it offered, it can't be declined here.

I'm not sure what you meant about altering the conditions for the other missions (assuming you meant missions past missions 7/7B), but I don't think `decline` would work because of the same reason, the missions were already accepted, so can't decline them afterwards.

> 		fail

Reviewer 4: Got it. what I meant be altering conditions for others was just updating everything so the relevant conditions were just replaced with `declined` instead of `failed`. But since this fix doesn't work here that's not relevant.

	to complete
		never

mission "Coalition: Expeditions Past 8"
	landing
	name "Lost Checkpoint"
	description "Travel to the marked systems so that Sedlitaris may look for a signal frequency of the expedition."
	passengers 1
	to offer
		has "Coalition: Expeditions Past 7: done"
	source "Watcher"
	waypoint "Rasalhague"
	waypoint "Cebalrai"
	waypoint "Zeta Aquilae"
	waypoint "Ascella"
	waypoint "Peacock"
	waypoint "Kaus Australis"
	on offer
		conversation

> 			`You land next to the forest again, and venture into it once more. The trail fresh in your memory, you two reach the clearing quickly, where Sedlitaris puts the pylon close to where it was before being dug out. "Almost like if we were there with them, this should be, being right where they left the pylon. Hopefully a better understanding of what they were thinking, it will also provide."`

Reviewer 4: ```suggestion
			`You land next to the forest again and venture into it once more. The trail fresh in your memory, you two reach the clearing quickly, where Sedlitaris puts the pylon close to where it was before being dug out. "Almost like if we were there with them, this should be, being right where they left the pylon. Hopefully a better understanding of what they were thinking, it will also provide."`
```

> 			`	She starts the recording, and sharp, static beeps are the first thing you hear. After a moment, you can make out agitated voices, as if the crew was discussing something, but Sedlitaris' translator doesn't pick it up. Finally, the familiar voice of Mimodum comes out, "Destroyed, the rest of the fleet was. Made it back to the system, one of the Punishers that headed north must have, ran into some debris pieces floating about in the system, we did. Take off, the rest of those on Summit couldn't. The ground is..." the static returns, this time sharing space with some scratch-like stops. "...cut like clay...rock...molten..." it switches to static once again, then stops for a moment. When it's back, the interference noises are gone. "Meet with the split group, I must. Warn them of what happened, and head back home with them."`

Reviewer 4: ```suggestion
			`	She starts the recording, and sharp, static beeps are the first thing you hear. After a moment, you can make out agitated voices, as if the crew was discussing something, but Sedlitaris' translator doesn't pick it up. Finally, the familiar voice of Mimodum comes out, "Destroyed, the rest of the fleet was. Made it back to the system, one of the Punishers that headed north must have; ran into some debris pieces floating about in the system, we did. Take off, the rest of those on Summit couldn't. The ground is..." the static returns, this time including some scratch-like stops. "... cut like clay... rock... molten..." It switches to static once again, then stops for a moment. When it's back, the interference noises are gone. "Meet with the split group, I must. Warn them of what happened, and head back home with them, I will."`
```

> 			`	You two look at the pylon in silence for a while, then she slowly approaches the machine to pick it up, saying "Supposed to head back home immediately, expedition ships were, if destroyed, one of the Punishers was. If followed those orders, he had... No guarantee of the split group's survival, he had. If back home, they went..."`

Reviewer 4: ```suggestion
			`	You two look at the pylon in silence for a while, then she slowly approaches the machine to pick it up, saying, "Supposed to head back home immediately, expedition ships were, if destroyed, one of the Punishers was. If followed those orders, he had... No guarantee of the split group's survival, he had. If back home, they went..."`
```

	on enter "Cebalrai"
		dialog `The pylon is quick to answer in this system, reporting no similar signals. Sedlitaris urges you to move to the next system, and goes back to checking the data.`
	on enter "Zeta Aquilae"
		dialog `Sedlitaris lets her pylon work for about an hour, but it returns no sign of any others in the system. "Passed by here, they might have, but a priority for recon, that world was not," she says as she points at Rand. You tell her you'll move on to the next system soon, and she goes back to working on the pylon.`
	on enter "Ascella"
		dialog `The pylon searches for some time, but tells Sedlitaris no others are in the system. She sighs lightly, and asks you to move on to the next.`
	on enter "Peacock"
		dialog `"If correct, my estimates were, the most likely meeting point, this should be," Sedlitaris says as you jump into the system. She lets the pylon work for a while, but it returns no signal of a similar device. "Well... either estimated wrong, I have, or issues in meeting here, the two groups had. Check the next system, we should."`
	on enter "Kaus Australis"

> 		dialog `After a couple of minutes, the pylon emits a few quiet whistles, and Sedlitaris celebrates briefly, "On that planet, it is! Glad that we managed to locate it, I am." You tell her you'll land on Hopper soon, so that you can search for the pylon on the planet.`

Reviewer 4: ```suggestion
		dialog `After a couple of minutes, the pylon emits a few quiet whistles, and Sedlitaris celebrates briefly. "On that planet, it is! Glad that we managed to locate it, I am." You tell her you'll land on Hopper soon so you can search for the pylon on the planet.`
```

	to fail
		has "Coalition: Expeditions Past 9: offered"
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris hasn't entered the system yet. Better depart and wait for it.`

mission "Coalition: Expeditions Past 9"
	landing
	name "Escape Route"
	description "Travel southwest of <system>, on a route to the Ablub system, so that Sedlitaris may look for a signal frequency of the expedition."
	passengers 1
	to offer
		has "Coalition: Expeditions Past 8: active"
	source "Hopper"
	on offer
		conversation
			`You fly around <origin> as Sedlitaris searches for the pylon's exact position, in the hopes that she can bring it back as well. She tells you to land by a mountain range, and has you readjust your landing spot every so often as she tries to nail the exact location of the pylon. "Buried it to protect it from the atmosphere's acidity, they must have," she comments.`
			`	By the time she's done, you open your ship's hatch right to where the pylon should be and help her dig it out, taking care to not be out for too long. When the device is dug out, you bring it inside your ship while Sedlitaris fills up the hole with what dirt had been dug out.`

> 			`	As you're flying to the hangars, she plays the pylon's entry: "Log started. Made a route from Summit straight to the meeting point, I had, but find the split group, I could not. Restless, agitated, the crew has become, heading straight to Ablub, we are, after buried, this thing is. If find this, anyone from the split group is, head back-" at that point an untranslated shouting muffles what Mimodum was saying, followed by dozens of legs in a rush. "No..." is heard right before the recording ends.`

Reviewer 4: `Hangars`? Where exactly are we going, Hopper's spaceport?

			`	You focus on flying the ship, but even when you've landed, Sedlitaris remains silent.`
			choice
				`	"Are you alright?"`
				`	"He said they were going back immediately, but they never made it. You think that was the last pylon?"`
				`	(Stay quiet).`

> 			`	She goes without a word for a while still. "Set a route for Ablub, could you, Captain? If follow the same route they used, we do, and if any more to find, there is, pick up the signals, I will."`

Reviewer 4: Could be a bit difficult to keep this mission updated on the off-chance any new planets on the way to Ablub are added later on that would change the automatic course the game plots. Unlikely, but food for thought.

Author: At that point the mission would need updating to keep up, yeah.

Reviewer 4: Since most of the game utilizes text replacements to make sure this isn't usually a problem, I'd think this could fall through the cracks if such a case occurs. I imagine we'd eventually discover the issue though. The only concern I have is if that route does change and Albub is no longer on it, then this whole ending might need to be revamped to accommodate that. I'm not too sure it's a good idea to merge with that in mind, but it's cool enough as-is that I'm torn. Gimme some time to chew on this one.

Reviewer 4: If worse comes to worse, I suppose we could add some line later on mentioning the specific route included Albub - that'd be a pretty quick fix that wouldn't involve revamping the whole ending. With that in mind, I'm good with this.

			`	You agree, and look to set a route straight to Ablub.`
				accept
	on enter "Girtab"
		dialog `While Sedlitaris remained in her bunk since you left for Ablub, you can hear her shouting from her bunk as you enter the system. She comes running to you, bringing along the pylon. "Picked up a signal, it has!" She says, pointing at the machine's panel, where some message is repeatedly beeping in and out. "Land on that planet we must, Captain!" You tell her you'll land soon, and she remains with you to figure out the pylon's location on Harmony once you've landed.`
	to complete
		never
	to fail
		has "Coalition: Expeditions Past 10: offered"
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris hasn't entered the system yet. Better depart and wait for it.`

mission "Coalition: Expeditions Past 10"
	landing
	name "Expeditions Past"
	description "Head back to the <destination>, where Sedlitaris will report her findings to the Heliarch council."
	passengers 1
	to offer
		has "Coalition: Expeditions Past 9: active"
	source "Harmony"
	destination "Ring of Friendship"
	on offer
		conversation

> 			`Once you're close to the planet's surface, Sedlitaris starts trying to access the recorded data in the pylon, "If possible, recover it later, we can." She tries for a few minutes, but all she can get is a twisted static, so she starts working on determining its location. "Clear up the transmission, getting closer to it should."`

Reviewer 4: ```suggestion
			`Once you're close to the planet's surface, Sedlitaris starts trying to access the recorded data in the pylon. "If possible, recover it later, we can." She tries for a few minutes, but all she can get is a twisted static, so she starts working on determining its location. "Clear up the transmission, getting closer to it should."`
```

			`	You fly around until you're right above an ocean, with no land in sight wherever you look. "Below us, it is," she says. "Why underwater? Why throw it into an ocean, would they?" She works on transferring the data to her own pylon, as you won't manage to get to the original one, and clears up the recording as much as possible before playing it.`

> 			`	For a few seconds there are only a few ambient sounds, a low background humming, and some heavy breathing. "Ambushed us as we prepared to leave the last planet, they did," Mimodum speaks. "Enough fuel to make it back in one go, we had, so jumped as we could. Too many ships in this system there were, however, like a wall blocking us." He stops briefly, and you can make out something akin to metal falling lightly on a table. "Aware of it, they were. That exploring, we had been. That splitting up, we were. That make it back, we could not. Saw to that, dozens of their ships did. Tried to lose them on this planet, we had, but caught as we came down, we were. Cut off most of the engines, skylance fire has. A lone, sinking ship, what remains of the sixth is. Hold together, the hull should, but no way out for us, there is." He pauses once again, for well over a minute. "If returned when the men first asked for... No, even then, already waiting, their ships must have been. If split up the fleet, I hadn't... outnumbered all the same, we would be. At least..." Another pause, this time just a few seconds long. "At least... still working, the rifles are."`

Reviewer 4: ```suggestion
			`	For a few seconds there are only a few ambient sounds, a low background humming, and some heavy breathing. "Ambushed us as we prepared to leave the last planet, they did," Mimodum speaks. "Enough fuel to make it back in one go, we had, so jumped as we could. Too many ships in this system there were, however, like a wall blocking us." He stops briefly, and you can make out something akin to metal falling lightly on a table. "Aware of it, they were. That exploring, we had been. That splitting up, we were. That make it back, we could not. Saw to that, dozens of their ships did. Tried to lose them on this planet, we had, but caught as we came down, we were. Cut off most of the engines, Skylance fire has. A lone, sinking ship, what remains of the sixth is. Hold together, the hull should, but no way out for us, there is." He pauses once again, for well over a minute. "If returned when the men first asked for... No, even then, already waiting, their ships must have been. If split up the fleet, I hadn't... outnumbered all the same, we would be. At least..." Another pause, this time just a few seconds long. "At least... still working, the rifles are."`
```
Skylance is _probably_ a proper noun.

			`	The recording ends, and you two stay in an awkward silence for a minute. "Aware that he had died, I was. Too long it has been. Even if found a safe haven, habitable planet or a friendly group, the fleet had, I knew that long dead, all of them would be." She pauses, and the silence returns for another minute. She shuts off the pylon. "Only, expecting something... anything, I was. A heroic death, a summary of findings. Accomplished nothing, they did. Right after all, the consuls were."`
			choice
				`	"Heroic death or not, he still got some information, didn't he? Information you recovered."`
					goto info

> 				`	"He didn't get the chance to act heroically, he was ambushed."`

Reviewer 4: ```suggestion
				`	"He didn't get the chance to act heroically. He was ambushed."`
```

			`	"Massacred, if truthful about the Quarg's numbers, he was," she responds. "Risked a great deal of jump drives, the sixth expedition had. Thought, it was, that with a greater number of Punishers, a greater chance of a safe return, there was." She sighs. "Just decrease our reserve of them, it did."`
				goto back
			label info
			`	"Inconclusive reports on red nebulas or disappearing ruins, what good are they?" she sighs. "After met up in their pincer movement, the groups had, after exploring points of interest, supposed to drive further than we had previously, they were. To reach the galactic core, to maybe find friendly groups to facilitate more trips... achieved, none of the goals were."`
			label back

> 			`	She goes back to her bunk, saying she wishes to be alone for a while, and you fly to the planet's hangars. A few hours later, she comes back out. "Finished my report, I have. Ready to go back home I am, Captain <last>. A long trip this was, and promised to pay you well, I did, so when <planet> we reach, give you your well earned 10,000,000 credits, I shall. Present my findings to the council, I must."`

Reviewer 4: ```suggestion
			`	She goes back to her bunk, saying she wishes to be alone for a while, and you fly to the planet's spaceport. A few hours later, she comes back out. "Finished my report, I have. Ready to go back home I am, Captain <last>. A long trip this was, and promised to pay you well, I did, so when <planet> we reach, give you your well-earned 10,000,000 credits, I shall. Present my findings to the council, I must."`
```
Gonna asssume you definitely mean spaceport this time.

			choice
				`	"That's a lot of money. I was worried you wouldn't even want to pay me, with how disappointed you were earlier."`

> 					goto alot

Reviewer 4: I get the label isn't in dialogue so spellcheck isn't especially relevant, but let's go ahead and change it so all your tests are passing.

				`	"Aren't you worried we'll get ambushed, like how Mimodum's ship was?"`
			`	She grins for a moment, seemingly taking it as a joke. "If going to ambush us, the Quarg were, already done it, they would have. Too close to the Coalition by now, we are."`
				goto accept
			label alot

> 			`	"Ah, forgive me for that, Captain. True it is, that disappointed, I am. However, an inspector should not let that trouble her duties," she explains. "Done your work, you have, so now do mine, I must. Not talked about in Mimodum's audio entries, many details of interest to the council were. An initial gathering of that data, I have prepared."`

Reviewer 4: ```suggestion
			`	"Ah, forgive me for that, Captain. True it is, that disappointed, I am. However, an inspector should not let that trouble her duties," she explains. "Done your work, you have, so now do mine, I must. Not discussed in Mimodum's audio entries, many details of interest to the council were. An initial gathering of that data, I have prepared."`
```

			`	"Inspector! A most... fortunate return this is. A great deal of information you've collected, we take it?" an Arach consul asks.`
			`	"Yes, eager to hear your report, we are," one of the Kimek consuls continues. "A rare occasion, it is, to be able to learn of the outside from one of our esteemed inspectors."`
			`	A Saryd consul, who's kept a stern look on his face, cuts in. "Returned with his fleet, your ancestor, Mimodum, did not. Learned of the reason for that, have you?"`
			`	"Yes," she says. "Recovered as much information as I wished to, I could not, but prepared a report on what me and Captain <last> found, I have. Additionally, recovered two of the pylons Mimodum left, we have."`
			`	One of the consuls adresses you, "Tell us, Captain, run into any er... complications during the trip, have you? Maybe occasioned by the tribune - the inspector - being with you?"`
			choice
				`	"She stayed inside my ship whenever there was a risk of humans seeing her, if that's what you're asking."`
				`	"I didn't notice anything out of the usual, no."`

> 			`	They call for some agents to get the pylons out of your ship, and transport them to somewhere Sedlitaris can present her findings. As they tell her to follow them to the room where the presentation will happen, a Kimek Heliarch breaks through the line of agents keeping the crowd back. You recognize him as Arbiter Kicharri, who you met just before Sedlitaris received approval from the Heliarch council.`

Reviewer 4: ```suggestion
			`	They call for some agents to get the pylons out of your ship and transport them to somewhere Sedlitaris can present her findings. As they tell her to follow them to the room where the presentation will happen, a Kimek Heliarch breaks through the line of agents keeping the crowd back. You recognize him as Arbiter Kicharri, who you met just before Sedlitaris received approval from the Heliarch council.`
```

			`	"What is this?" The Saryd consul steps in front of Kicharri as he approaches. "Arbiter, shared with you in due time, the findings will be, but the inspector's report to us consuls, the priority is, so return to your duties you-"`
			`	"No, meet with her family, she will," Kicharri interrupts him. "Heard nothing but hell from her brother, I have. Devastated, her parents were. Now that returned, she has, see them and calm their worries, she will."`
			`	"By whose orders?" An Arach consul jumps in. "Intrigued by her findings, we all are, and listen to her at once, we must."`
			`	"Orders?" Kicharri exclaims. "Orders for her to leave and die, you thought you had given her!" Kicharri looks at Sedlitaris, then back at the consuls. "If so intrigued you are, surely to miss out on any information, you don't want. Best suited for giving a full report, she will be, once rested from her expedition, she has."`

> 			`	Irked by his response, the consuls have the agents clear the area, and have your ship made into a makeshift meeting room for a while, making everyone get out while they head inside to discuss. After many minutes, they come back out, and one of the Kimek says, "Ten days of leave you will be given, tribune. After that, present us your findings, you must. Considered for promotion to arbiter, you also will be, for your bravery and work in the expedition."`

Reviewer 4: ```suggestion
			`	Irked by his response, the consuls have the agents clear the area and have your ship made into a makeshift meeting room for a while, making everyone get out while they head inside to discuss. After many minutes, they come back out, and one of the Kimek says, "Ten days of leave you will be given, tribune. After that, present us your findings, you must. Considered for promotion to arbiter, you also will be, for your bravery and work in the expedition."`
```

> 			`	The consul group leaves, and Kicharri approaches you and Sedlitaris, "Tell me about how you were right all along, later, you can," Kicharri tells Sedlitaris. "Right now, add you back to the list of active Heliarchs, I must. Marked as deceased by a 'glitch in the system,' you apparently were." He taps one leg on the ground repeatedly, glancing at your ship. "Glad that you are back safely, I am."`

Reviewer 4: ```suggestion
			`	The consul group leaves, and Kicharri approaches you and Sedlitaris. "Tell me about how you were right all along, later, you can," Kicharri tells Sedlitaris. "Right now, add you back to the list of active Heliarchs, I must. Marked as deceased by a 'glitch in the system,' you apparently were." He taps one leg on the ground repeatedly, glancing at your ship. "Glad that you are back safely, I am."`
```

			choice
				`	"Well, we made it back safely, and you were passionate enough about this to take the risk. I don't hold it against you."`
					goto forgive
				`	"So you lied to me. You could've gotten us both killed - we could be in a ship sinking into an ocean."`
			`	She's taken aback. "Captain, no Quarg pursuing us, there were. Surely now that inhabit all those systems, humans do, avoid attacking ships like that..." She stops, and sighs. "Sorry for lying to you I am, Captain."`
				goto end
			label forgive
			`	She breathes in relief. "Thank you, Captain. To trick you, my intention wasn't. Thanks to your help, maybe put an end to this fear of leaving, or even convince the council to get the expedition efforts going again, I can."`
			label end

> 			`	Suddenly, a Saryd Sojourner nearly rams your ship, just barely avoiding it as it comes to a stop while turning. "My little brother," Sedlitaris says as she looks at the ship. "Forgive me Captain, but get to seeing my family, I must. From the discussion earlier, very worried about me, they must have been. Thanks again for everything." Her brother rushes into hugging her as he leaves his ship, then proceeds to shake her while screaming something. An elderly Saryd couple slowly comes out of the Sojourner as well, and Sedlitaris and her brother go and meet them, hugging and going inside. The Sojourner takes off before the spaceport workers can even begin tending to the ship, and you look at it as it leaves the ringworld.`

Reviewer 4: ```suggestion
			`	Suddenly, a Saryd Sojourner nearly rams your ship, just barely avoiding it as lands less twenty meters from your ship. "My little brother," Sedlitaris says as she looks at the ship. "Forgive me Captain, but get to seeing my family, I must. From the discussion earlier, very worried about me, they must have been. Thanks again for everything." Her brother rushes into hugging her as he leaves his ship, then proceeds to shake her while screaming something. An elderly Saryd couple slowly comes out of the Sojourner as well, and Sedlitaris and her brother go and meet them, hugging and going inside. The Sojourner takes off before the spaceport workers can even begin tending to the ship, and you look at it as it leaves the ringworld.`
```

			`	"Repilor, a work trip, this is. Away for long, I won't be, so wish to trouble you all with the news, I didn't," she responds.`
			`	"Trouble us? In truth, know that wrong, you are, but face telling father that you're throwing your life away, you couldn't!"`
			`	He tries to push his way to her, but the guards stop him before he can make much progress. One of the Heliarch representatives gets closer, saying, "A restricted area, this is. Invading government property, you are, so suggest that you leave immediately, I do."`
			`	"Restricted? What about the outside? Is it not also restricted? Much more dangerous? Fine with sending my sister there, you seem," Repilor counters, and the Heliarch looks away after a moment. "If bothering the consuls so much, she was, just fired, she should've been!" He then looks at you, and continues, "And the human? Your transport out of here, I imagine? Or maybe the one who put these ideas in your head, they are?"`
			choice
				`	"I've just been hired to transport her."`
					goto hired
				`	"What's so wrong about her wanting to explore outside the Coalition?"`
					goto wrong
				`	"Wait, what did you say about the outside being dangerous?"`

> 			`	The Saryd looks at you, puzzled. Then, as if realizing something, turns to Sedlitaris, "Told the human, you haven't?"`

Reviewer 4: ```suggestion
			`	The Saryd looks at you, puzzled. Then, as if realizing something, he turns to Sedlitaris. "Told the human, you haven't?"`
```

> 			`	"That's enough Repilor. Trespassing on a restricted area, you are. Escort him to his ship," she tells the guards, and after exchanging looks briefly, and after some struggle on Repilor's part, the Arachi manage to push him back inside the Sojourner. "Ignore that you should, Captain. Always... difficult, my brother was. Too stubborn to realize when to stop, and easy to impress with folk tales of the outside."`

Reviewer 4: ```suggestion
			`	"That's enough Repilor. Trespassing on a restricted area, you are. Escort him to his ship," she tells the guards, and after some struggle on Repilor's part, the Arachi manage to push him back inside the Sojourner. "Ignore that you should, Captain. Always... difficult, my brother was. Too stubborn to realize when to stop, and easy to impress with folk tales of the outside."`
```

			choice
				`	"If you say so. Should I wait in my ship until you all are done here?"`
					goto accept
				`	"Folk tales or not, I'm not really willing to find out after that. He didn't seem to be lying."`
			`	"What?" she exclaims. "Please Captain, fall for my brother's little rant, you cannot. Finally permission to leave, I have!"`
			choice
				`	"Alright, alright. But I hope I don't regret this."`
				`	"I'm sorry. Whatever it is he was afraid of, I'm not willing to risk it."`
					goto decline
			`	"You won't." She says, heading to you and holding your hands with hers, as if in thanks. "Promise you I do, fine we will be, Captain."`
				goto accept
			label decline
			`	You hear her stomping her hoofs on the ground as you head back to your ship, and see her and the Heliarch council representatives arguing with one another as you close your hatch. When you're back on the spaceport landing area, a Sojourner lands next to your ship, and Repilor, the Saryd from earlier, comes out.`

> 			`	"Forgive me for being rude earlier, you must." He says as he greets you. "Thought I had, that talked my sister into this you had. That filled her head with more grandiose tales of the outside, you did." He sighs. "Thank you, Captain. Hate you now, my sister must, but the right thing, you did. Better for the both of you, this is." He nods lightly in a bow, and heads back to his ship.`

Reviewer 4: ```suggestion
			`	"Forgive me for being rude earlier, you must." He says as he greets you. "Thought I had, that talked my sister into this you did. That filled her head with more grandiose tales of the outside, you must have." He sighs. "Thank you, Captain. Hate you now, my sister must, but the right thing, you did.
			`	"That no inspector has ever returned from their expedition, the truth is, so afraid you were leading my sister to her death, I was. Better for the both of you, this is." He nods lightly in a bow, and heads back to his ship.`
```
Probably should have some payoff for the `"Told the human, you haven't?"` line here. You could even add an option to ask Sedlitaris about what he was talking about later and have her just sort of brush it off, but not necessary in that case since the player will learn at the end anyway. Feel free to edit the suggestion as you see fit though.

Author: I'm not sure on giving a choice to call Sedlitaris out on that line from Repilor, since I'd need some good deal more label branching to make sure it only showed up if you picked the choices that make Repilor say that.
Added a bit at the end of this line to try for a payoff, let me know if you meant something other than what I did.
